core-bundle/contao/languages/en/default.xlf

```diff
@@ -1074,13 +1074,13 @@
         <source>Fit the box</source>
       </trans-unit>
       <trans-unit id="MSC.box.1">
-        <source>The image will be resized proportionally so that both sides do not exceed the given dimensions. Same as &lt;code&gt;contain&lt;/code&gt; in CSS.</source>
+        <source>The image will be resized proportionally so that both sides do not exceed the given dimensions. Same as &lt;em&gt;contain&lt;/em&gt; in CSS.</source>
       </trans-unit>
       <trans-unit id="MSC.crop.0">
         <source>Crop (important part)</source>
       </trans-unit>
       <trans-unit id="MSC.crop.1">
-        <source>Preserves the important part of an image as specified in the file manager. If necessary, the image will be cropped.</source>
+        <source>Preserves the important part of an image as specified in the file manager. If necessary, the image will be cropped. Same as &lt;em&gt;cover&lt;/em&gt; in CSS.</source>
```

Comment:
I think, important point is missing: `The image will be resized to the exact dimensions given. The image will be cropped if necessary, preserving the important part (if specified in the file manager).`